devops/openshift/database/mongo-ha-param.yml

```diff
@@ -1,7 +1,7 @@
 MEMORY_REQUEST=256Mi
-MEMORY_LIMIT=1Gi
+MEMORY_LIMIT=2Gi
 CPU_REQUEST=250m
 CPU_LIMIT=0.5
-VOLUME_CAPACITY=1Gi
+VOLUME_CAPACITY=2Gi
```

Comment:
Was it increased based on some recommendations? 
We will not be using Mongo to store anything else but form definitions, right?

Reply:
yes, i increased it, as we had 2GB last time in our old mongo volume. So changed the yml this time after checking the old voume.

Reply:
Ok, but still do not answer the questions.
- "Was it increased based on some recommendations?", apparently no.

Old Mongo volume from FormFlow solution was expecting to have the Mongo receive all the submitted form data.
It does not seem a big deal to keep 2GB, but it seems that there is no reason to do so 😉